android/app/src/main/java/app/organicmaps/util/StringUtils.java

```diff
@@ -19,6 +19,11 @@ public static String formatUsingUsLocale(String pattern, Object... args)
     return String.format(Locale.US, pattern, args);
   }
 
+  public static String formatUsingSystemLocale(String pattern, Object... args)
+  {
+    return String.format(Locale.getDefault(), pattern, args);
```

Comment:
Locale.getDefault() doesn't handle dynamic switching of languages in the system. It would be better to use context.getResources().getConfiguration().getLocales().get(0). Nevertheless, this issue can be dealt separately across the code base.